frontend/src/static/js/v_authorship.js

```diff
@@ -184,6 +184,26 @@ window.vAuthorship = {
       }
     },
 
+    getFileBlameLink(file) {
+      const repo = window.REPOS[this.info.repo];
+
+      return `http://github.com/${
+        repo.location.organization}/${
+        repo.location.repoName}/blame/${
+        repo.branch}/${
+        file.path}`;
```

Comment:
check jslint, not sure if this is the right format, dun look very pretty hahahhaahh